apisix/core/response.lua

```diff
@@ -110,6 +116,31 @@ function _M.set_header(...)
 end
 
 
+function _M.add_header(...)
```

Comment:
`add_header` is almost same with `set_header`, we should extract common logic to avoid repeat code.

Reply:
@ShiningRush 
Updated.

Reply:
nice ^_^